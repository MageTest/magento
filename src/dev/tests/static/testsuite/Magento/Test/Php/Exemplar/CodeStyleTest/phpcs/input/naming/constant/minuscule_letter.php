<?php
define('SOME_CONSTaNT_2_ACT_4_YOU', 42);
define("SOME_CONSTaNT_2_ACT_4_YOU", 2783);
class Magento_Test_Php_Exemplar_CodeStyleTest_phpcs_input_naming_constant_minuscule_letter
{
    const SOME_LONG_CONSTaNT_2_ACT_4_YOU = 1;
}
