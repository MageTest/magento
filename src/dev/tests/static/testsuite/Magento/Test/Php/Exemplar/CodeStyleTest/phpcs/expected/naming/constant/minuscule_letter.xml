<?xml version="1.0" encoding="UTF-8"?>
<config>
    <error line="2"/>
    <error line="3"/>
    <error line="6"/>
</config>
